packages/core/workers/src/Worker.js

```diff
@@ -48,13 +49,23 @@ export default class Worker extends EventEmitter {
       v => !/^--(debug|inspect)/.test(v)
     );
 
-    let options = {
+    this.child = childProcess.fork(childModule, process.argv, {
       execArgv: filteredArgs,
       env: process.env,
-      cwd: process.cwd()
-    };
+      cwd: process.cwd(),
+      stdio: ['ignore', 'pipe', 'pipe', 'ipc']
+    });
 
-    this.child = childProcess.fork(childModule, process.argv, options);
+    this.child.stdout.on('data', data => {
```

Comment:
Are these chunked into lines automatically by node or could the chunks be shorter/longer than a line?

Reply:
Good question. Afaict, Node doesn't chunk anything; newlines exist in the stream because they're written there by console.log (which puts them in automatically) or writing to stdout directly with a newline at the end.

Splitting by newline definitely assumes that new lines map to new log messages. I'm not 100% about this since e.g. `console.log('foo\nbar')` would get treated as two separate messages.

Conversely, Node buffers writable streams and is inconsistent about when it flushes `data` events, which is why this splitting exists in the first place -- before splitting, tests would fail inconsistently because the stream would emit a `data` event containing more than one logical message.

I can look a bit more into this.